src/components/TextInput/styles.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+
+export const InpuText = styled.input`
+  width: 300px;
+  height: 35px;
+  outline: none;
+  margin: 0;
+  margin-left: 5px;
+  padding: 6px;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  background: #ffffff;
```

Comment:
The file `styles/GlobalStyles.js` contains CSS variables of the colors used in the project as other basic styles setup. Please check it

Reply:
How You use the vars with the styled-componets ? when add background: var(--primary-color); in the template literary didn't change